addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/ApplicationErrorScanRuleUnitTest.java

```diff
@@ -63,6 +63,7 @@ public void setUpZap() throws Exception {
                         + "-->\n"
                         + "<Patterns>\n"
                         + "  <Pattern type=\"string\">Microsoft OLE DB Provider for ODBC Drivers</Pattern>\n"
+                        + "  <Pattern type=\"string\">SyntaxError: Unexpected token</Pattern>\n"
```

Comment:
Better test with the file.

Reply:
For the purposes of the UnitTests this is the file, unless you expect me to completely rewrite all of this?

Reply:
IMO this test class should be testing just the scan rule not the copied patterns in the bundled file, we should have a specific test class using the bundled file.

Reply:
So I shouldn't add a UnitTest for this PR?

Reply:
You should just not on this test class, tests of the default patterns should be tested separately.

Reply:
Oh okay, I think I see what you’re getting at.

Reply:
Is there another rule with UnitTests already doing that, so I don't have to re-invent the wheel?

Reply:
The closer would be `ApplicationErrorScanRuleRegexMatcherUnitTest`.

Reply:
Sorry I was anticipating having to do something special to get the file read properly, based on past discussions, but it works out-of-the-box so to speak.